### ISSUE TITLE
check whether `safeCPickleDumps(obj):` is still needed

### ISSUE THREAD

**Issue by sbillinge** [diffpy/diffpy.pdfgui#189]
This may be another py2 compatibility function.  Check where it is used and whether it can be removed.

Write tests to make sure the behavior is still handled.

**Comment by Tieqiong** [comment]
@sbillinge this method ensures that an object is serialized even if it contains problematic floating-point values by falling back to a simpler serialization protocol if the default one fails. 

In Python 3, the pickle module uses more recent [protocols](https://docs.python.org/3/library/pickle.html#data-stream-format), and the `SystemError` related to `frexp()` is less likely to occur. So I think the fallback to ASCII protocol (protocol 0) is not necessary any more.

Yes test would be the right thing to do

**Comment by bobleesj** [comment]
@Tieqiong will take care of it for v3.1.0

**Comment by bobleesj** [comment]
@Tieqiong could you please share an update on this issue?

**Comment by sbillinge** [comment]
@Tieqiong can we close this?

**Comment by Tieqiong** [comment]
Yes let's close this one! @sbillinge @bobleesj  thanks

**Comment by sbillinge** [comment]
@Tieqiong we need a report on what you did and why it is ok to close it.

**Comment by sbillinge** [comment]
yes, we want to keep this open then.  But I will move it to a later milestone.   No hurry on this.
